### DISCUSSION TITLE
Error code '400'

### DISCUSSION THREAD

**Discussion by garbe90** [djdd87/SynoAI#127] [Q&A]
Hi, I tried to set up synoai with docker following the guide but every time I start the container it hangs right after it's launched. I always receive this error in the logs: Login: Failed due to Synology API error code '400'. I'm using my admin account and I also tried to create another user with full access on surveillance station but nothing works. Can somebody help me?
Thanks

**Comment by djdd87** [comment]
Did you follow the FAQ to the letter and create the user from **within** Surveillance Station, or did you just create a new user from DSM and give it permission to SS?

This issue seems to occur when you use a user created from DSM and the workaround is to create the user from SS itself from the Admin > Users menu.

![image](https://user-images.githubusercontent.com/22151993/168807178-789c65e0-e7e3-479e-91ac-22deb8983d55.png)

If this isn't the case, then unfortunately I've not got any other suggestions as it's not a SynoAI issue.

**Reply by garbe90** [reply]
Thanks for the quick reply. I've just found the problem, user needs to have a password without symbols. In my case the password had some # and for some reason, it wasn't recognized as correct.

**Reply by djdd87** [reply]
Brilliant! Glad you've got it working.

**Comment by djdd87** [comment]
This is now fixed by #128
